Client/Frontend/Browser/BrowserLocationView.swift

```diff
@@ -132,13 +138,23 @@ class BrowserLocationView : UIView, UIGestureRecognizerDelegate {
     var url: NSURL? {
         didSet {
             lockImageView.hidden = (url?.scheme != "https")
-            if let url = url?.absoluteString {
-                if url.hasPrefix("http://") ?? false {
-                    locationLabel.text = url.substringFromIndex(advance(url.startIndex, 7))
-                } else if url.hasPrefix("https://") ?? false {
-                    locationLabel.text = url.substringFromIndex(advance(url.startIndex, 8))
+
+            if let urlString = url?.absoluteString {
+
+                var truncatedURL: String
+                if let scheme = url?.scheme {
+                    truncatedURL = urlString.substringFromIndex(advance(urlString.startIndex, count(scheme) + count("://")))
```

Comment:
Cool - I know I before we were stripping out the https:// but I can change it so that we only strip out http:// but keep https:// to show the user we're displaying a https site